README.md

```diff
@@ -13,7 +13,3 @@ This extension activates automatically when project's `tsconfig.json` file has `
 - TypeScriptToLua diagnostics
 
 ![](/docs/diagnostics.png)
-
-- Custom options schema for `tsconfig.json` file
-
-![](/docs/tsconfig-schema.png)
```

Comment:
The plugin still adds tsconfig validation though, the schema just moved. I propose we keep this but just update the link to point to the main repo

Reply:
Ok, I updated the docs with a better explanation about what the feature does exactly